### PR TITLE
Callhome changes

### PR DESCRIPTION
**Pull request by NicoZelaya**
We are removing the variable `PORT` because it's too common. Prefixing environment variables with SPLITIO.

Adding support for multiple keys on get-treatments endpoint. Filtering responses by traffic type.

> - ADD support for getTreatments?keys= [ { "matchingKey: "..", "bucketingKey" : "(this is optional)", "trafficType" : "…"}, {…} ]
> - REMOVE support for getTreatments?key= ` (singular key)
> - keep `getTreatment?key= ` (key singular)
> - REMOVE getTreatments?…..&split-names
> - when getTreatments is called (all flags), only return the one for the traffic type(s) specified. The > rest of the split without matching traffic type are ignored and not included in the response.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 1. `nvm use`
 2. `npm install`
-3. `API_KEY=xxxxxxx PORT=4444 SPLITIO_DEBUG='on' npm start`
+3. `API_KEY=xxxxxxx SPLITIO_DEBUG='on' npm start`
```
Comment (patricioe) [inline README.md]: doesn't `SPLITIO_SERVER_PORT` apply here?

Comment (NicoZelaya) [reply README.md]: Yes, you can use it on Node alone as well. I'll add it back.

File: server.js [hunk 2]
```diff
@@ -71,47 +74,77 @@ app.get('/get-treatment', (req, res) => {
...
+
 app.get('/get-treatments', (req, res) => {
   const state = req.query;
-  const key = utils.parseKey(state.key, state['bucketing-key']);
+  let keys = [];
+  try {
+    keys = reduce(JSON.parse(state.keys), (acc, e) => {
+      acc.push(e);
+      return acc;
+    }, []);
```
Comment (marciocaraballo) [inline server.js]: You could use map here instead of reduce, since you are returning everything

File: server.js [hunk 3]
```diff
@@ -71,47 +74,77 @@ app.get('/get-treatment', (req, res) => {
...
+
+  const splitsPromise = Promise.resolve(manager.splits()).then(views => {
+    let filteredSplits = [];
+    keys.forEach(key => {
+      filteredSplits.push({
+        trafficType: key.trafficType,
+        key: utils.parseKey(key.matchingKey, key.bucketingKey),
+        splits: filterSplitsByTT(views, key.trafficType)
+      });
+    });
```
Comment (marciocaraballo) [inline server.js]: I think you can use map here also

File: server.js (hunk 2 above)
Comment (NicoZelaya) [reply server.js]: I'll just remove the loop as I'm not doing processing here.

File: server.js (hunk 3 above)
Comment (NicoZelaya) [reply server.js]: Changed.

**Pull request merged**